www-misc/profile-sync-daemon/profile-sync-daemon-6.28.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="Symlinks and syncs browser profile dirs to RAM."
+HOMEPAGE="https://wiki.archlinux.org/index.php/Profile-sync-daemon"
+
+SRC_URI="https://github.com/graysky2/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+RESTRICT="mirror"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+RDEPEND="
+	app-shells/bash
+	net-misc/rsync[xattr]
+	sys-apps/systemd"
+
+pkg_postinst() {
+	local replacing
+	for replacing in ${REPLACING_VERSIONS}; do
+		if [[ "$(get_major_version ${replacing})" == "5" ]]; then
```

Comment:
`get_major_version` is provided by the versionator eclass, so it should be inherited properly.